mars_api/models.py

```diff
@@ -80,5 +78,5 @@ class PlayerScore(models.Model):
 
     def __str__(self):
         return f"""nickname: {self.player.nickname},
-        game date: {self.game.date},
-        corporation: {self.corporation} """
+        game_id: {self.game_id}, game date: {self.game.date},
+        corporation: {self.corporation}."""
```

Comment:
I'm very sure you'd like to look into this in more detail since I doubt that it will come out like you expect.

Reply:
Also keep in miind this string will come out in the dropdowns (if there's any place this model would be a candidate for it). Also think about where will it get used mostly - newlines may not makes sense in that context.

Reply:
![image](https://user-images.githubusercontent.com/34490649/89731522-95d85900-da3f-11ea-9db0-32818c21b0bf.png)
seems to be working?

Reply:
Seems to be be working (take 2 :clapper: )
![image](https://user-images.githubusercontent.com/34490649/89736821-3476b100-da64-11ea-9406-72289793653e.png)

Reply:
Just check whether you're sure that you want to control *how they look* in your backend?

Reply:
I thought it would be nice when debugging/using interactive shell. A convenient method for identifying an object. Is it harmful do have a custom `__str__` method? I am not sure what problems could it cause?